docs/how-to-guides/running-feast-in-production.md

```diff
@@ -77,16 +99,35 @@ In the above example we are materializing the source data from the `driver_hourl
 
 The timestamps above should match the interval of data that has been computed by the data transformation system.
 
-## 3. How to use Feast for model training and serving
+### 2.1. Automate periodic materializations 
 
-Now that you have deployed a registry, provisioned your feature store, and loaded your data into your online store, your clients can start to consume features for training and inference.
+It is up to you which orchestration / scheduler to use to periodically run `$ feast materialize`.
+However, since Feast keeps history of materialization in its registry the choice could be as simple as unix cron util
+(if you have just one machine for this job). More complicated cases can be handled with Kubernetes Jobs or Airflow.
```

Comment:
| (if you have just one machine for this job)

Seems a bit cryptic.

| More complicated cases can be handled with Kubernetes Jobs or Airflow.

What would be an example of a more complicated case?